plugins/gestures.koplugin/main.lua

```diff
@@ -466,24 +468,55 @@ Default value: %1]]), Screen.low_pan_rate and 5.0 or 30.0),
                 end,
                 separator = true,
             },
+            {
+                text = _("Tap interval"),
+                keep_menu_open = true,
+                callback = function()
+                    local SpinWidget = require("ui/widget/spinwidget")
+                    local GestureDetector = require("device/gesturedetector")
+                    local items = SpinWidget:new{
+                        title_text = _("Tap interval"),
+                        info_text = T(_([[
+Another tap made in this interval after a first tap will be considered a bounce and ignored.
+
+The interval value is in milliseconds and can range from 0 (0 second) to 2000 (2 seconds).
+Default value: %1]]), GestureDetector.TAP_INTERVAL/1000),
+                        width = math.floor(Screen:getWidth() * 0.75),
+                        value = GestureDetector:getInterval("ges_tap_interval")/1000,
+                        value_min = 0,
+                        value_max = 2000,
+                        value_step = 50,
+                        value_hold_step = 200,
+                        ok_text = _("Set interval"),
+                        default_value = GestureDetector.TAP_INTERVAL/1000,
+                        callback = function(spin)
+                            G_reader_settings:saveSetting("ges_tap_interval", spin.value*1000)
+                            GestureDetector:setNewInterval("ges_tap_interval", spin.value*1000)
+                        end
+                    }
+                    UIManager:show(items)
+                end,
+            },
             {
                 text = _("Double tap interval"),
+                keep_menu_open = true,
                 callback = function()
                     local SpinWidget = require("ui/widget/spinwidget")
                     local GestureDetector = require("device/gesturedetector")
                     local items = SpinWidget:new{
-                        text = T(_([[
-Set double tap interval in milliseconds.
-The interval value can range from 100 (0.1 seconds) to 2000 (2 seconds).
+                        title_text = _("Double tap interval"),
+                        info_text = T(_([[
+When double tap is enabled, this sets the time to wait for the second tap. Only after this interval has ellapsed will a single tap be emitted.
```

Comment:
elapsed, but maybe something like
```suggestion
When double tap is enabled, this sets the time to wait for the second tap. A single tap will take at least this long to be detected.
```